README.md

```diff
@@ -75,6 +77,10 @@ Second, use the `pip` command from the previous step to install the package in t
 1. Make sure you can build the project by doing all of the `pip` steps above
 2. In PyCharm, run the _Unit Tests_ configuration.
 
+## Known Issues
```

Comment:
I think you're conflating two known issues:

1. Snowflake cannot write timezone'd times to Parquet (AOBS-491; this PR). The README should explain the consequence of our work around (that timezone information is dropped, potentially link to SF articles about it)
2. Dataiku can only read date times as `int96` rather than the more standard annotated Logical Types. You can simply say `Dataiku Parquet issues documented in #12`, or make no mention of it as its outside the scope of this PR.